common/app/views/fragments/commentBox.scala.html

```diff
@@ -21,13 +21,24 @@
         </div>
     </div>
 
+    <div class="d-comment-box__onboarding">
+        <h3>Welcome, you’re about to make your first comment!</h3>
```

Comment:
That was on the original spec in fact! So definitely a good idea. I'll look to re-add it.